build.gradle

```diff
@@ -26,18 +26,9 @@ repositories {
 dependencies {
     constraints {
         // Libraries
-        api('com.exactpro.th2:common:2.6.0')
-        api('com.exactpro.th2:sailfish-utils:2.2.0')
-        api('com.exactpro.th2:check1:2.2.0')
-        api('com.exactpro.th2:check2:2.2.0')
-        api('com.exactpro.th2:store-common:2.6.0')
-
         api("com.rabbitmq:amqp-client:5.9.0")
         api("com.datastax.cassandra:cassandra-driver-core:3.6.0")
```

Comment:
Does the cradle use this BOM? If it doesn't probably we should remove this dependency from here and leave it only in the cradle